src/combineActions.js

```diff
@@ -0,0 +1,28 @@
+import isString from 'lodash/isString';
+import isFunction from 'lodash/isFunction';
+import isEmpty from 'lodash/isEmpty';
+import toString from 'lodash/toString';
+import isSymbol from 'lodash/isSymbol';
+
+export const ACTION_TYPE_DELIMITER = '|redux-action-delimiter|';
```

Comment:
I haven't seen it using https://github.com/gaearon/redux-devtools for a few months in production. I don't think it'd come up using https://github.com/evgenyrodionov/redux-logger.

Actions are still dispatched per-type; it's just that registering them with a library reducer is different.